eclipse-collections-code-generator/src/main/resources/api/map/mutableObjectPrimitiveMap.stg

```diff
@@ -38,6 +39,17 @@ public interface MutableObject<name>Map\<K> extends Object<name>Map\<K>
 
     void put(K key, <type> value);
 
+    /**
+     * This method allows MutableObject<name>Map the ability to add an element in the form of Object<name>Pair\<K>.
+     *
+     * @see #put(Object, <type>)
+     * @since 9.1.0
+     */
+    default void putPair(Object<name>Pair\<K> keyValuePair)
```

Comment:
This should return the value to be consistent with put and other putPair implementations.

Reply:
No you cannot return a value. Since the value is a primitive, what do I return when the key does not exist in map?
This is consistent with put in mutableObjectPrimitiveMap

Reply:
Ok, that makes sense.  Thanks for clarifying.